### ISSUE TITLE
Latest stable protoc version

### ISSUE THREAD

**Issue by pdebakker-auguria** [arduino/setup-protoc#99]
### Describe the problem

The readme states
```
To get the latest stable version of protoc just add this step:

- name: Install Protoc
  uses: arduino/setup-protoc@v3
```
But this installs version 23.4, while the latest version is 26.0.

When I look at the default version value in action.yml, I see it is set to 23.x.
Is it possible to update this version, and maybe even somehow always get the latest value without having to hardcode it?

### To reproduce

use `arduino/setup-protoc@v3`
this gives protoc with version 23.x

### Expected behavior

Version 26.0 of protoc

### 'arduino/setup-protoc' version

3

### Issue checklist

- [X] I searched for previous reports in [the issue tracker](https://github.com/arduino/setup-protoc/issues?q=)
- [X] I verified the problem still occurs when using the latest version
- [X] My report contains all necessary details

**Comment by alessio-perugini** [comment]
@pdebakker-auguria Absolutely! Right now, I cannot guarantee that we have enough bandwidth to work on this in the upcoming days. :pray: 
If you or someone else would like to contribute to this would be awesome! :star_struck: 

Regardless of who will implement this, I'll live here some implementation notes:
- Change default to `latest` https://github.com/arduino/setup-protoc/blob/master/action.yml#L7
- If the version variable is `latest` implement the logic to retrieve and use the latest versions. We should change it here https://github.com/arduino/setup-protoc/blob/master/src/installer.ts#L210-L239
- Add a dedicated test

**Comment by adombeck** [comment]
Hi! It's sad to see this GitHub Action is not well maintained anymore. `arduino/setup-protoc@v3` still installs protoc v23.4, while the README claims that it installs the latest stable release (which would be v32 now).

I also see that the last time a PR was merged or commented on was in 2024. So before I start preparing a PR with the required changes and a test: Would any of the maintainers still be up for reviewing and merging such a PR?
